dev_scripts/env.py

```diff
@@ -322,20 +324,7 @@ def __init__(self, distro_name, distro_version):
     @property
     @functools.lru_cache
```

Comment:
This can be removed now, I think

Reply:
... and I should use a proper docstring. Thanks for catching this, I have a fixup in ce39cfe071afc0e693040f03e316e69bb8e9b6b5

Reply:
Your fixup seems to have ended up on https://github.com/freedomofpress/dangerzone/pull/779 instead of this PR, but it LGTM.

Reply:
Ouch. I will properly squash the commits, thanks for flagging this.